components/log/include/esp_log.h

```diff
@@ -86,6 +86,21 @@ vprintf_like_t esp_log_set_vprintf(vprintf_like_t func);
  */
 uint32_t esp_log_timestamp(void);
 
+/**
+ * @brief Function which returns system timestamp to be used in log output
+ *
+ * This function is used in expansion of ESP_LOGx macros to print
+ * the system time as "HH:MM:SS.sss". The system time is initialized to
+ * 0 on startup, this can be set to the correct time with an SNTP sync,
+ * or manually with standard POSIX time functions.
+ *
+ * Currently this will not get used in logging from binary blobs
+ * (i.e WiFi & Bluetooth libraries), these will still print the RTOS tick time.
```

Comment:
I'll check if we can resolve this atsome point in the future.

Reply:
Yeh, essentially the problem was the print format for those log messages was still expecting an integer, not a string, so all the logging from the wifi binary was just printing the memory address of the buffer as an integer. If there is a way you could get the binaries to be compatible that would be great.